index.rst

```diff
@@ -147,176 +246,121 @@ The white light source and monochromator sit on a table surrounded by sturdy bla
 The linear actuator that drives the shutter sits on the top of the structure and must be disconnected before the top is removed. 
 
 
-Setup
-=====
+Maintenance & Troubleshooting
+=============================
 
-Mechanical and Electrical Assembly
-----------------------------------
-There are three crates that were shipped to Chile from Tucson that contain all necessary hardware for the AuxTel Calibration Illumination System. The list of what was shipped in which crate can be found on `this confluence page <https://confluence.lsstcorp.org/display/LTS/AuxTel+Illumination+System>`__.
+Opening Electronics Cabinet
+---------------------------
 
-Follow this general list of activties to assemble the AuxTel Illumination System
+In order to open the electronics cabinet, you must first power down the unit. Then undo the two screw tabs that keep it closed.
```

Comment:
Please add that the computer should be powered down before opening the cabinet.

Reply:
done